voicer-client/src/App.js

```diff
@@ -11,30 +14,113 @@ import { DataContext } from "./context/DataContext"
 import "./App.scss"
 
 function App() {
+
+  //Material UI Dark Mode
+  const prefersDarkMode = useMediaQuery('(prefers-color-scheme: dark)')
+
+  const theme = React.useMemo(
```

Comment:
let's see if we can clean this up before hand off, separate this into a different file for readability sake